src/CommunityToolkit.Maui/Alerts/Toast/Toast.shared.cs

```diff
@@ -0,0 +1,93 @@
+﻿using CommunityToolkit.Maui.Core;
+
+namespace CommunityToolkit.Maui.Alerts;
+
+/// <inheritdoc/>
+public partial class Toast : IToast
+{
+	/// <inheritdoc/>
+	public string Text { get; set; } = string.Empty;
+
+	/// <inheritdoc/>
+	public ToastDuration Duration { get; set; }
+
+	/// <summary>
+	/// Create new Toast
+	/// </summary>
+	/// <param name="message">Toast message</param>
+	/// <param name="duration">Toast duration</param>
+	/// <returns>New instance of Toast</returns>
+	public static IToast Make(
+		string message,
+		ToastDuration duration = ToastDuration.Short)
+	{
+		return new Toast()
```

Comment:
Let's add a null check here, just in case a user does have Nullable enabled.

```suggestion
		ArgumentNullException.ThrowIfNull(message);
		
		return new Toast()
```

Reply:
Let's then also add a Unit Test to verify this:

```cs
[Fact] 
public void ToastMake_NewToastCreatedWithNullString_ShouldThrowArgumentNullException() 
{ 
    #pragma warning disable CS8625 // Cannot convert null literal to non-nullable reference type.
    Assert.Throws<ArgumentNullException>(() => Toast.Make(null));
    #pragma warning restore CS8625 // Cannot convert null literal to non-nullable reference type.
}
```